.github/PULL_REQUEST_TEMPLATE/default.md

```diff
@@ -0,0 +1,40 @@
+---
+labels: Needs Review
+name: Pull Request
+---
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
```

Comment:
We typically link to a issue with a `Closes #`, `Fixes #, or `Part of #`.

Reply:
Are you suggesting that we have that as a prompt in the template?